Content.Server/Silicons/Laws/IonStormSystem.cs

```diff
@@ -0,0 +1,278 @@
+using Content.Server.StationEvents.Components;
+using Content.Shared.Administration.Logs;
+using Content.Shared.Database;
+using Content.Shared.Dataset;
+using Content.Shared.FixedPoint;
+using Content.Shared.GameTicking.Components;
+using Content.Shared.Random;
+using Content.Shared.Random.Helpers;
+using Content.Shared.Silicons.Laws;
+using Content.Shared.Silicons.Laws.Components;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+using System.Linq;
+
+namespace Content.Server.Silicons.Laws;
+
+public sealed class IonStormSystem : EntitySystem
+{
+    [Dependency] private readonly IPrototypeManager _proto = default!;
+    [Dependency] private readonly ISharedAdminLogManager _adminLogger = default!;
+    [Dependency] private readonly SiliconLawSystem _siliconLaw = default!;
+    [Dependency] private readonly IRobustRandom _robustRandom = default!;
+
+    // funny
+    [ValidatePrototypeId<DatasetPrototype>]
+    private const string Threats = "IonStormThreats";
+    [ValidatePrototypeId<DatasetPrototype>]
+    private const string Objects = "IonStormObjects";
+    [ValidatePrototypeId<DatasetPrototype>]
+    private const string Crew = "IonStormCrew";
+    [ValidatePrototypeId<DatasetPrototype>]
+    private const string Adjectives = "IonStormAdjectives";
+    [ValidatePrototypeId<DatasetPrototype>]
+    private const string Verbs = "IonStormVerbs";
+    [ValidatePrototypeId<DatasetPrototype>]
+    private const string NumberBase = "IonStormNumberBase";
+    [ValidatePrototypeId<DatasetPrototype>]
+    private const string NumberMod = "IonStormNumberMod";
+    [ValidatePrototypeId<DatasetPrototype>]
+    private const string Areas = "IonStormAreas";
+    [ValidatePrototypeId<DatasetPrototype>]
+    private const string Feelings = "IonStormFeelings";
+    [ValidatePrototypeId<DatasetPrototype>]
+    private const string FeelingsPlural = "IonStormFeelingsPlural";
+    [ValidatePrototypeId<DatasetPrototype>]
+    private const string Musts = "IonStormMusts";
+    [ValidatePrototypeId<DatasetPrototype>]
+    private const string Requires = "IonStormRequires";
+    [ValidatePrototypeId<DatasetPrototype>]
+    private const string Actions = "IonStormActions";
+    [ValidatePrototypeId<DatasetPrototype>]
+    private const string Allergies = "IonStormAllergies";
+    [ValidatePrototypeId<DatasetPrototype>]
+    private const string AllergySeverities = "IonStormAllergySeverities";
+    [ValidatePrototypeId<DatasetPrototype>]
+    private const string Concepts = "IonStormConcepts";
+    [ValidatePrototypeId<DatasetPrototype>]
+    private const string Drinks = "IonStormDrinks";
+    [ValidatePrototypeId<DatasetPrototype>]
+    private const string Foods = "IonStormFoods";
+
+    //Randomly alters the laws of an individual silicon.
```

Comment:
```suggestion
    /// <summary>
    /// Randomly alters the laws of an individual silicon.
    /// </summary>
```
This way the summary shows up in the IDE if you hover over the function with your mouse.